### ISSUE TITLE
Error installing pyautogui

### ISSUE THREAD

**Issue by LiverpoolRed** [LifeAlgorithm/VitalSourcePrinter#4]
Hi,
Thanks for the hard work you've done creating this tool.

I am on a mac. I installed python 3.6 & PyPDF2 with no issues. However when I try installing pyautogui I get the following error:
"python setup.py egg_info" failed with error code 1 in /private/var/folders/bm/k674gvpd5x3f0p05v4gxm7b80000gn/T/pip-build-u9_d7dd8/pyautogui/

I have tried installing Pillow then the pyautogui but I still get the error.

I am not good at coding. Can you please tell me how to fix this?

thanks

**Comment by LifeAlgorithm** [comment]
Hi, let's see if this works. Try running "pip3 install --upgrade pip" then running "pip3 install Pillow" (it may already be added from last step), and then "pip3 install pyautogui".  

If the above doesn't work, try "sudo easy_install Pillow" then "pip3 install pyautogui"

**Comment by LiverpoolRed** [comment]
The above didn't work. After some searching, I installed pyobjc prior to installing pyautogui. Once done I was able to install pyautogui. Now I need to find a pdfprinter for mac.

Just to be sure I had quartz installed, I downloaded it and installed it before installed the pyobjc

Process was:
pip3 install --upgrade pip
pip3 install PyPDF2
pip3 install Pillow
pip3 install pyobjc --core
pip3 install pyobjc
pip3 install pyautogui
